<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FlyEmSettingDialog</class>
 <widget class="QDialog" name="FlyEmSettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FlyEM - Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="toolTip">
          <string>Set configuration file. Effective only when 'Use Default' is OFF.</string>
         </property>
         <property name="text">
          <string>Config File</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="defaultConfigLineEdit">
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
         <property name="frame">
          <bool>false</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="configLineEdit">
         <property name="toolTip">
          <string>Set configure</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="configPushButton">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="defaultConfigFileCheckBox">
       <property name="toolTip">
        <string>Use default configuration file (distributed with the program)</string>
       </property>
       <property name="text">
        <string>Use Default</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Remote Service</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="servicelineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>--Status:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="statusLabel">
         <property name="text">
          <string>Normal</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Task Server</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="taskServerLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QCheckBox" name="profilingCheckBox">
       <property name="text">
        <string>Profiling</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="autoStatuscCheckBox">
       <property name="text">
        <string>Auto status check</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Verbose</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="verboseSpinBox">
       <property name="maximum">
        <number>6</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>DVID:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="parallelTileCheckBox">
       <property name="text">
        <string>Parallell tile fetching</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="prefetchCheckBox">
       <property name="text">
        <string>Lowtis Prefetching</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Data Direcotry</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="dataDirLineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="dataDirPushButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QCheckBox" name="synapseNameCheckBox">
       <property name="text">
        <string>Naming Synapses</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="psdNameCheckBox">
       <property name="text">
        <string>Naming PSD</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Mesh Split</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="meshThreSpinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>999</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>x1M</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Center Cut:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Gray</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="grayCxSpinBox">
       <property name="maximum">
        <number>9999</number>
       </property>
       <property name="value">
        <number>256</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>x</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="grayCySpinBox">
       <property name="maximum">
        <number>9999</number>
       </property>
       <property name="value">
        <number>256</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>Seg</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="segCxSpinBox">
       <property name="maximum">
        <number>9999</number>
       </property>
       <property name="value">
        <number>256</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>x</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="segCySpinBox">
       <property name="maximum">
        <number>9999</number>
       </property>
       <property name="value">
        <number>256</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Note: You must click the 'Apply' button to apply new settings.</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="updatePushButton">
       <property name="toolTip">
        <string>Apply current settings</string>
       </property>
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closePushButton">
       <property name="statusTip">
        <string>Close the dialog. All settings modified without the update will be discarded.</string>
       </property>
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
